### ISSUE TITLE
Deploy without deno deploy

### ISSUE THREAD

**Issue by locothedev** [denoland/fresh#583]
Hey! I'd like to know if there is anyone who has deployed his fresh app without deno deploy and can kindly share an example on how to do so

**Comment by fulsomenko** [comment]
I used a dockerfile like this:
```
FROM denoland/deno:alpine-v1.23.0

EXPOSE 8000

WORKDIR /app

USER deno

COPY . .

RUN deno cache main.ts --import-map=import_map.json

CMD ["run", "--allow-net", "--allow-env", "--allow-read", "main.ts"]
```

**Comment by locothedev** [comment]
@fulsomenko thanks mate! Will test this later appreciate you 🥇

**Comment by excessivecoding** [comment]
Don't forget to include `DENO_DEPLOYMENT_ID` in your image as Fresh uses this env variable to determine if it's running in production or not.

Related issue: https://github.com/denoland/fresh/issues/535

**Comment by rojvv** [comment]
This article might be useful, too, for Linux machines with systemd:


https://blog.jlcarveth.dev/hello_world

**Comment by lucacasonato** [comment]
If someone wants to write docs for this, I'd be happy to merge a PR :)

**Comment by lino-levan** [comment]
@lucacasonato This should be closed right (with #680 being merged)?
